litex_boards/platforms/sipeed_tang_nano_4k.py

```diff
@@ -88,10 +88,10 @@ class Platform(GowinPlatform):
 
     def __init__(self, toolchain="gowin"):
         # Educational GoWin IDE v. 1.9.8.03
-        GowinPlatform.__init__(self, "GW1NSR-LV4CQN48PC6/I5", _io, _connectors, toolchain=toolchain, devicename="GW1NSR-4C")
+        GowinPlatform.__init__(self, "GW1NSR-LV4CQN48PC6/I5", _io, _connectors, toolchain=toolchain, devicename="GW1NS-4")
         # Licensed GoWin IDE v. 1.9.7.06 Beta
         # GowinPlatform.__init__(self, "GW1NSR-LV4CQN48PC7/I6", _io, _connectors, toolchain=toolchain, devicename="GW1NSR-4C")
-        self.toolchain.options["use_mode_as_gpio"] = 1
+        #self.toolchain.options["use_mode_as_gpio"] = 1
```

Comment:
Somehow we don't have this setting in Apicula, or maybe it's named differently, to be investigated.

Reply:
Why not, in a first time, using a test to add this option?
- with gowin's toolchain the current behavior is left unchanged
- with apicula's toolchain this option is not added and we have to verify/fix result ;-)

Reply:
Possible but first I'd like to look into why we don't have this option. Either we have it under a different name, or we're missing something.

Reply:
It's just a software check we don't have so we can make it conditional on the toolchain